api/internal/handler/nft_handler.go

```diff
@@ -171,7 +188,7 @@ func createNftRecord(uri string, tokenId int, ownerAddress string, collectionAdd
 		return nil, fmt.Errorf("failed to split uri string to 2 parts. uri: %v", uri)
 	}
 
-	decodedMetadata, err := base64.URLEncoding.DecodeString(splittedStr[1])
+	decodedMetadata, err := base64.StdEncoding.DecodeString(splittedStr[1])
```

Comment:
why this change?

Reply:
The URLEncoding failed to decode some of the URIs in the NFT contract. But StdEncoding works for all tokens.